lib/agent0/agent0/hyperdrive/exec/run_agents.py

```diff
@@ -158,7 +158,7 @@ def setup_agents(
         contract_addresses = fetch_hyperdrive_address_from_uri(os.path.join(eth_config.artifacts_uri, "addresses.json"))
 
     # create hyperdrive interface object
-    interface = HyperdriveInterface(
+    interface = HyperdriveReadWriteInterface(
```

Comment:
I wonder if we should just keep the name here to be `HyperdriveInterface` to avoid verboseness